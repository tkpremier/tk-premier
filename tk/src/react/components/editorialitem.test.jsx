import React from 'react';
import EditorialItem from './editorialitem';
import renderer from 'react-test-renderer';

 it('should render an editorial item', () => {
    const editorialitem = renderer.create(
        <EditorialItem
            htmlCaption="Feature Video | Photographs"
            imageUrl="https://8675fa3b12fbaae97df1-49af760ec3cce3b92480c70a3569b570.ssl.cf2.rackcdn.com/HomePage/featuredContent-186.jpg"
            url="https://www.phillips.com/Video/default?VideoID=190"
        />
    );
    const tree = editorialitem.toJSON();
    expect(tree).toMatchSnapshot();
});
