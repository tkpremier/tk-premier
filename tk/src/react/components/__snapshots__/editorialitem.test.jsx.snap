// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an editorial item 1`] = `
<li
  className={undefined}
>
  <a
    href="https://www.phillips.com/Video/default?VideoID=190"
  >
    <div
      className="content-wrapper"
    >
      <div
        className="image-container"
        style={
          Object {
            "backgroundImage": "url(https://8675fa3b12fbaae97df1-49af760ec3cce3b92480c70a3569b570.ssl.cf2.rackcdn.com/HomePage/featuredContent-186.jpg)",
            "backgroundPosition": "50% 50%",
            "backgroundSize": "cover",
          }
        }
      />
      <span
        className="caption"
        dangerouslySetInnerHTML={
          Object {
            "__html": "Feature Video | Photographs",
          }
        }
      />
    </div>
  </a>
</li>
`;
